Sources/BraintreeShopperInsights/BTCreateCustomerSessionApi.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// The API used to create a customer session using the `CreateCustomerSession` GraphQL mutation.
+/// - Warning: This feature is in beta. It's public API may change or be removed in future releases.
+class BTCreateCustomerSessionApi {
```

Comment:
I'm not familiar with visibility modifiers in swift, but is this Api class exposed to merchants? Should we ensure it's not visible?